rosidl_typesupport_freertps_cpp/cmake/rosidl_typesupport_freertps_cpp_generate_interfaces.cmake

```diff
@@ -281,6 +170,8 @@ add_dependencies(
 )
 
 if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
+  ament_export_libraries(${rosidl_generate_interfaces_TARGET}${_target_suffix} ${freertps_LIBRARIES})
+  ament_export_include_directories(include)
```

Comment:
I recommend to keep this line close to the lines which install the header files.